src/main/java/com/conveyal/datatools/manager/controllers/api/FeedSourceController.java

```diff
@@ -277,9 +278,12 @@ public static FeedSource requestFeedSourceById(Request req, String action) {
         return requestFeedSource(req, FeedSource.get(id), action);
     }
     public static FeedSource requestFeedSource(Request req, FeedSource s, String action) {
```

Comment:
As there're no tests, could you either : add some doc to understand changes, or add a test ?